src/Makefile.am

```diff
@@ -188,6 +190,10 @@ libsub_root_a_SOURCES = \
 	thread_base.cc \
 	thread_base.h \
 	thread_worker.cc \
-	thread_worker.h
+	thread_worker.h \
+	\
+	vendor/tinyxml2/tinyxml2.h \
+	vendor/tinyxml2/tinyxml2.cpp
```

Comment:
Don't like using vendor directory here, better to use `rpc/tinyxml2/tinyxml2.h` and `namespace rpc::tinyxml2` imo.